mozilla/fx-private-relay
# Add Premium Onboarding

Fixes MPP-1087

Related L10N String PR: https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/14

TO-DO: 
- [x] Add l10n Strings
- [x] Style UI for onboarding dismissal 
- [x] Add logic to only show to premium users
- [x] Add JS/dismissal functionality
- [x] Edgecase: Premium account with ZERO aliases (double onboarding) 

## Testing steps: 

Test 1: Premium Visible/Dismissal
- Log in with premium account
- **Expected:** Premium onboarding is visible on dashboard page
- Click on dismiss text on bottom of premium onboarding
- **Expected:** Premium onboarding is no longer visible
- Close browser/window and open dashboard page again or refresh tab
- **Expected:** Premium onboarding is not visible

_Edgecase: If a user is premium and has NO aliases created, the premium onboarding should not be visible. (The standard onboarding should be). After creating at least one, the premium onboard is visibel (and the inital onboarding is not)._

Test 2: Non-premium 
- Log in with non-premium account
- **Expected:** Premium onboard is not visible
